### ISSUE TITLE
Wrong output

### ISSUE THREAD

**Issue by AlexWayfer** [greyblake/crystal-icr#9]
When I compile file with code:
```crystal
puts "5a".to_i
```
I don't get errors on building and get exception on executing:
```
invalid Int32: 5a (ArgumentError)
[4348647] *CallStack::unwind:Array(Pointer(Void)) +87
[4348538] *CallStack#initialize<CallStack>:Array(Pointer(Void)) +10
[4348490] *CallStack::new:CallStack +42
[4348383] *Exception +31
[4348317] *ArgumentError#initialize<ArgumentError, String>:CallStack +29
[4348257] *ArgumentError::new<String>:ArgumentError +97
[4346497] *String#to_i32<String, Int32, Bool, Bool, Bool, Bool>:Int32 +257
[4346224] *String#to_i<String, Int32, Bool, Bool, Bool, Bool>:Int32 +32
[4346172] *String#to_i<String>:Int32 +92
[4322853] ???
[4330153] main +41
[140684254578192] __libc_start_main +240
[4279081] _start +41
[0] ???
```
But now `icr`:
```crystal
icr(0.12.0) > "5a".to_i

icr(0.12.0) >
```

**Comment by greyblake** [comment]
Thanks for the report, looking...

**Comment by greyblake** [comment]
That is fixed in 5a32801
